tensorflow/tools/ci_build/linux/rocm/rocm_py37_pip.sh

```diff
@@ -42,8 +42,18 @@ export N_TEST_JOBS=$(expr ${TF_GPU_COUNT} \* ${TF_TESTS_PER_GPU})
 # Get the default test targets for bazel.
 source tensorflow/tools/ci_build/build_scripts/DEFAULT_TEST_TARGETS.sh
 
+#Add filter tags specific to ROCm version
+rocm_major_version=`cat /opt/rocm/.info/version | cut -d "." -f 1`
+rocm_minor_version=`cat /opt/rocm/.info/version | cut -d "." -f 2`
+TF_TEST_FILTER_TAGS_ROCM_VERSION_SPECIFIC=""
+if [[ $rocm_major_version == *"5"* ]]; then
```

Comment:
I think we also need a check for $rocm_major_version -lt 5

Reply:
Thanks for catching this. Fixed!